utils/rpms/daos.spec

```diff
@@ -146,6 +147,7 @@ Requires: ndctl
 # needed to set PMem configuration goals in BIOS through control-plane
 %if (0%{?suse_version} >= 1500)
 Requires: ipmctl >= 03.00.00.0423
+Requires: pciutils
```

Comment:
This is superfluous with line 162.

```suggestion
```

Reply:
@tanabarr Please don't resolve conversations I start.  I use them to validate my outstanding requests and will resolve them when I see that they are fixed correctly.